great-idea/services.html

```diff
@@ -1,107 +1,142 @@
+                    tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur
+                    feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce
+                    finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod
+                    facilisis iaculis.</p>
+                    <button>Learn More</button>
+                  </div>
+                  <div class="text-content">
+                    <h4>Learn More Digital Marketing</h4>
+                    <p>Nulla suscipit interdum
+                      diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet,
+                      iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur
+                      cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh
+                      pellentesque eleifend. Mauris euismod facilisis iaculis.</p>
+                      <button>Learn More</button>
+                  </div>
+                  <div class="text-content">
+                    <h4>Learn More Web
+                      Development</h4>
+                    <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus
+                      rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat
+                      justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus
+                      sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod
+                      facilisis iaculis.</p>
+                      <button>Learn More</button>
+                  </div>
+                  <div class="text-content">
+                    <h4>Learn More iOS Development</h4>
+                    <p>Nulla suscipit interdum diam
+                      at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis
+                      tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et
+                      justo eu consequat. Fusce finibus sem neque, sed faucibus nibh
+                      pellentesque eleifend. Mauris euismod facilisis iaculis.</p>
+                      <button>Learn More</button>
+                  </div>
+                  <div class="text-content">
+                    <h4>Learn More
+                      Android Development</h4>
+                      <p>Nulla suscipit interdum diam at maximus. Proin vitae
+                        tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur
+                        feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce
+                        finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod
+                        facilisis iaculis.</p>
+                        <button>Learn More</button>
+                  </div>
+          </div>
+                
+          
+          
+          
+            
+          </div>
```

Comment:
again I would put these closing divs up higher and make sure the nesting looks good